### ISSUE TITLE
Track ownership status of aircraft

### ISSUE THREAD

**Issue by n8rzz** [openscope/openscope#1223]
We need to track ownership so we can render when an a/c is: 
- owned by you
- in handoff status to you
- in handoff status from you
- or is owned by someone else 

This first step will include adding the property and methods to support ownership and changing ownership:
- [ ] spawned arrivals -> in handoff to you
- [ ] aircraft within airspace -> owned by you
- [ ] departures out of airspace -> in handoff from you

**note: the current owner will not be rendered as part of this task.  changes here are all under the hood.**

_(part of Controller Handoffs Epic #1221 )_

**Comment by erikquinn** [comment]
Being within the airspace does not necessarily mean we own the track. "Owned by us" would be whenever we accept the handoff. Owning the track does not mean we have the right to turn/climb/descend the aircraft, but it does mean we can edit the data block. Only when we have track control and the a/c is inside our airspace do we have the right to start doing stuff to them.

Departures out of the airspace we will have to initiate a handoff to center. For the time being, I think we can have handoffs to center award points, but not penalize when users don't (because they may not know they're supposed to).

**Comment by n8rzz** [comment]
Erik, you're not thinking 4th dimensionally...

This issue is just to get things moving, all the handoffs here will happen programmatically and won't mean anything until #1226 is implemented.

**Comment by felixscheffer** [comment]
How would `RadarTargetModel` fit in? `RadarTargetModel` already has a property called `_isUnderOurControl` which we don't use at the moment. Maybe we can expand that into an emum that tracks the current ownership status.

**Comment by n8rzz** [comment]
I like that idea!  That seems like the most appropriate place for this logic to live, since we are dealing with the `RadarTargetModel` and not necessarily the `AircraftModel` itself.

**Comment by erikquinn** [comment]
Exactly, I was trying to say that it specifically belongs there in `RadarTargetModel._isUnderOurControl`, and is not determined by checking `.isWithinAirspace()`, for example. That's why it was placed there before it was even used :smiley:

However, as I think about it, there is a much more intelligent way to handle it: replace that property with `._controllingSector` and `._handoffSector`, and add methods `.hasTrackControl(sectorModel)`, `.isHandoffStatus()`, `.isHandoffStatusToSectorModel(sectorModel)`, and `.acceptHandoff()`. Whoever owns the track is the controlling sector. When a handoff is initiated, that sector becomes the handoff sector. When handoff sector has a value, we do the blinky thing to show them in handoff status. If the handoff is cancelled, we remove the value from handoffSector. If it was accepted, we set controllingSector to handoffSector, and clear the handoff sector.
